packages/common-grpc/src/service.js

```diff
@@ -178,6 +179,11 @@ function GrpcService(config, options) {
 
   this.grpcMetadata = new grpc.Metadata();
 
+  this.grpcMetadata.add('x-goog-api-client', [
+    'gl-node/' + process.versions.node,
+    'gccl/' + (config.packageJson.version || ''),
```

Comment:
I don't think we'll run into a case where `version` isn't defined, unless you did run into this?